### ISSUE TITLE
Ability to quit an app when no windows

### ISSUE THREAD

**Issue by sohate** [burakssen/QuitMe#2]
Hey! Can you please add an ability to close an app when it has no open window. Thanks!

**Comment by burakssen** [comment]
@sohate Hi, I wanted to ask, do you mean background applications?

**Comment by sohate** [comment]
@burakssen Hi, no, only apps with active windows (something like Stocks, Podcast and etc. (not exactly these apps)).

**Comment by burakssen** [comment]
@sohate Are these apps like windowless? Can you give me an example application so that I can test.

**Comment by sohate** [comment]
@burakssen These apps have active windows, but when all the windows are closed, these apps still active in dock, and it will be great if some app (your for example) can quit them if there is no active windows.

Apps like these: Apple stocks, Apple podcasts, Apple Pages, Apple Numbers, Apple Mail, Apple Notes. 3rd party: qBittorrent

**Comment by burakssen** [comment]
@sohate I think this already works 🤔 I mean If I understand correctly. When you click close button on top of the windows macOS basically don't close the applications completely right. On my application they still shown as running applications and you can quit them as well.

**Comment by sohate** [comment]
@burakssen Yes, but it would be great if you app can do it automatically when all windows are closed.

**Comment by burakssen** [comment]
@sohate Oh ok, now I get it, sorry. That would be a good feature, Maybe I can add some kind of timer based closing system for specified windowless apps?

**Comment by sohate** [comment]
@burakssen nah, you shouldn't be sorry, it's my fault, i though your app have auto close feature. Timer it's a good idea if it would be support seconds (like 1 or 5 seconds). Because there is apps with Timer feature but no one has "seconds".

**Comment by burakssen** [comment]
@sohate Ok I'll think about this and implement it for future releases.

**Comment by Hydro8** [comment]
Yes it could be a great feature !
